### ISSUE TITLE
Cannot type into text input on jqModal dialog when modal set to true

### ISSUE THREAD

**Issue by mgroves** [briceburg/jqModal#13]
I recently upgraded jqModal from r14 to r24.

I'm experiencing the same issue described in this Stack Overflow question: http://stackoverflow.com/questions/29395113/cannot-type-into-text-input-on-jqmodal-dialog-when-modal-set-to-true and the same fix (comment out modal:true).

However, since we are on r24, I'm confused, since your Stack Overflow answer indicates that it was fixed in r23.

Additionally, this changes the functionality of the software I'm writing, since clicking outside the modal didn't previously close the modal. This may or may not be a big deal to the product owners, but I wanted to open this issue to hopefully get some more information before approaching them about it.

Thanks,

Matt

**Comment by spetryjohnson** [comment]
This should be closed - the issue was with our custom code wrapping jqModal, not jqModal itself.

Turns out we had a piece of code that was clearing and resetting the CSS class list for the popup window div. This worked fine in older versions of jqModal, but r23 added a "jqm-init" class that is used by the code that manages the focus event. Since we were inadvertently removing that class, we were  ultimately creating an endless focus loop. 

Once we started preserving the "jqm-init" CSS class everything worked perfectly.
